conans/client/rest/auth_manager.py

```diff
@@ -221,10 +231,18 @@ def authenticate(self, user, password):
                 user = prev_user
 
         try:
-            token = self._rest_client.authenticate(user, password)
+            token, refresh_token = self._rest_client.authenticate(user, password)
         except UnicodeDecodeError:
             raise ConanException("Password contains not allowed symbols")
 
+        logger.debug("Got token")
```

Comment:
Seems a bit weird these debug traces.

Reply:
Why? The debug traces help debug what is going on, right?

Reply:
Anyway. Removed.